sdk/storage/Azure.Storage.Blobs/tests/BlobClientTests.cs

```diff
@@ -541,6 +538,35 @@ async Task Verify(Task<Response<BlobContentInfo>> upload)
             }
         }
 
+        private static async Task DownloadAndAssert(long size, Stream stream, BlobClient blob)
+        {
+            var actual = new byte[Constants.DefaultBufferSize];
+            using var actualStream = new MemoryStream(actual);
+
+            // reset the stream before validating
+            stream.Position = 0;
+
+            // we are testing Upload, not download: so we download in partitions to avoid the default timeout
+            for (var i = 0; i < size; i += Constants.DefaultBufferSize * 5 / 2)
```

Comment:
Why `* 2.5`?

Reply:
This was already there. I think it is for performance as we are not checking every partition of the blob.